code/game/objects/items/weaponry.dm

```diff
@@ -586,6 +586,34 @@
 	flags_1 = DROPDEL_1 | ABSTRACT_1
 	attack_verb = list("bopped")
 
+/obj/item/slapper
+	name = "slapper"
+	desc = "This is how real men fight."
+	icon_state = "latexballon"
+	force = 1
+	throwforce = 0
+	flags_1 = DROPDEL_1 | ABSTRACT_1
+	attack_verb = list("slapped")
+	hitsound = 'sound/effects/snap.ogg'
+
+/obj/item/slapper/attack(mob/M, mob/living/carbon/human/user)
+	if(user.a_intent != INTENT_HARM)
+		var/aim_for_face = ((user.zone_selected == "mouth") || (user.zone_selected == "eyes"))
+		user.do_attack_animation(M)
+		playsound(M.loc, 'sound/weapons/slap.ogg', 50, 1, -1)
+		user.visible_message("<span class='danger'>[user] slaps [M] in the [(aim_for_face)?"face":user.zone_selected]!</span>",
+ 		"<span class='notice'>You slap [M] in the [(aim_for_face)?"face":user.zone_selected]! </span>",\
+ 		"You hear a slap.")
+		if(ishuman(M))
+			var/mob/living/carbon/human/L = M
+			L.endTailWag()

```

Comment:
Is this supposed to be here?

Reply:
Yeah

Reply:
its copypasted code from slapping feature

It was originally made to stop lizards wagging their tails.